docs/references/api/tables_views.rst

```diff
@@ -1142,3 +1101,19 @@ Using ``offset`` to target a different subset of rows is also possible.
 
   There is no native ``UPDATE...LIMIT`` or ``DELETE...LIMIT`` support in PostgreSQL; the generated query simulates that behavior and is based on `this Crunchy Data blog post <https://www.crunchydata.com/blog/simulating-update-or-delete-with-limit-in-postgres-ctes-to-the-rescue>`_.
 
+.. raw:: html
+
+  <script type="text/javascript">
+    let hash = window.location.hash;
+
+    const redirects = {
+      // Tables and Views
+      '#computed-virtual-columns': 'api/computed_fields.html#computed-fields',
```

Comment:
I think this is failing to redirect: https://postgrest--650.org.readthedocs.build/en/650/references/api/tables_views.html#computed-virtual-columns

Maybe this works:

```suggestion
      '#computed-virtual-columns': 'computed_fields.html#computed-fields',
```

---

Also, I think maybe these lines should be changed in the `api.rst` file to avoid unnecessary redirects (the result is the same):

https://github.com/steve-chavez/postgrest-docs/blob/6ccf695435853f6283a5b4295cec56a1f482c3ff/docs/references/api.rst?plain=1#L38

```javascript
      '#computed-virtual-columns': 'api/computed_fields.html#computed-fields',
```

Reply:
Redirect works now :raised_hands: